docs/pbc/all/dynamic-multistore/202311.0/base-shop/dynamic-multistore-feature-overview.md

```diff
@@ -80,3 +80,58 @@ Normal storefront APIs can be used to fetch data from stores using the Store HTT
 The following image shows the ability to call API endpoints with the exemplary "SECOND" store selected in the API call itself.
 
 ![storefront-api](https://spryker.s3.eu-central-1.amazonaws.com/docs/pbc/all/dynamic-multistore/dynamic-multistore.md/storefront-api.png)
+
```

Comment:
I have doubts regarding 202311.0 version. 
Apparently, this should be discussed with the PO to align the next release date and move the updated doc there or get the confirmation that we leave it as it is. 
But as I see we have a fresher version for the doc under 202404.0

Reply:
You are right. Moved to 202404.0